.github/workflows/gh-pages.yaml

```diff
@@ -11,24 +11,49 @@ on:
     paths:
       - 'book/**'
 
+env:
+  CARGO_TERM_COLOR: always
+  # Update the language picker in index.hbs to link new languages.
+  LANGUAGES: zh
+
 jobs:
   deploy:
-    runs-on: ubuntu-20.04
```

Comment:
Hmm... I see:

![image](https://user-images.githubusercontent.com/5236035/235668690-209ad22f-ce3e-4a5b-9283-27e406a09e2f.png)

Does it mean, after merging, the master branch will fail to deploy?

Reply:
I didn't get it. where did you see it? Is it something I can't access?

Reply:
I see - probably because I am using Refined GitHub (an open source plugin)

![image](https://user-images.githubusercontent.com/5236035/235678643-1784b8ae-7997-4977-894b-8f5f48a8f88e.png)

Reply:
> My understanding is that the previous deployment process was broken, and after I modified gh-pages.yaml, the deployment process should have worked, but since I didn't modify the contents of the book folder, it didn't trigger the automatic deployment.

Then maybe trigger the deployment s.t. we are 100% sure ;)
e.g. commit to change one newline; then commit to change it back

Reply:
> I see - probably because I am using Refined GitHub (an open source plugin)

doesn't work for me, but I think it refers to this page https://github.com/fzyzcjy/flutter_rust_bridge/deployments